src/program.ts

```diff
@@ -3408,6 +3438,9 @@ export class Class extends TypedElement {
   /** Tests if a value of this class type is assignable to a target of the specified class type. */
   isAssignableTo(target: Class): bool {
     var current: Class | null = this;
+    if (target.kind == ElementKind.INTERFACE) {
+       return true;
```

Comment:
This looks wrong?

Reply:
This is because checking the interface requires resolving function prototypes.  Which is why the check happens in the compiler method.  This was an artifact as the condition in the compiler's method checks if the target is an interface so this never actually gets triggered.  That is my least favorite part of this solution since it should happen in this method.